hack/e2e/README.md

```diff
@@ -0,0 +1,51 @@
+# Eventing End-to-End Tests
+Tests the end-to-end flow for Eventing manager.
+
+## Overview
+
+This test covers the end-to-end flow for Eventing. It tests the creation of an Eventing CR and verifies that all the required resources are provisioned by the Eventing manager. It also tests the end-to-end delivery of events for different event types.
+
+## Usage
+
+### Pre-requisites
+
+- [Go](https://go.dev/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- Access to Kubernetes cluster ([k3d](https://k3d.io/) / k8s)
+- Eventing manager deployed to the cluster without any Eventing CR.
+- For EventMesh backend, the following secrets must exist in the cluster:
```

Comment:
Do you have a command to set these two secrets?

Reply:
We need to use the script to generate the secret, as we do in release-testing.